sdk/identity/Azure.Identity/README.md

```diff
@@ -165,19 +138,11 @@ var blobClient = new BlobClient(
 
 ### Define a custom authentication flow with `ChainedTokenCredential`
 
-While the `DefaultAzureCredential` is generally the quickest way to get started developing applications for Azure, more advanced users may want to customize the credentials considered when authenticating. The `ChainedTokenCredential` enables users to combine multiple credential instances to define a customized chain of credentials. This example demonstrates creating a `ChainedTokenCredential` which will attempt to authenticate using managed identity, and fall back to authenticating via the Azure CLI if managed identity is unavailable in the current environment. The credential is then used to authenticate an `EventHubProducerClient` from the [Azure.Messaging.EventHubs][eventhubs_client_library] client library.
-
-```C# Snippet:CustomChainedTokenCredential
-// Authenticate using managed identity if it is available; otherwise use the Azure CLI to authenticate.
-
-var credential = new ChainedTokenCredential(new ManagedIdentityCredential(), new AzureCliCredential());
-
-var eventHubProducerClient = new EventHubProducerClient("myeventhub.eventhubs.windows.net", "myhubpath", credential);
-```
+While `DefaultAzureCredential` is generally the quickest way to authenticate apps for Azure, you can customize the credentials that are considered. `ChainedTokenCredential` enables users to combine multiple credential instances to define a customized chain of credentials. For more information, see [ChainedTokenCredential overview][ctc_overview].
```

Comment:
When I initially read the first sentence it sounded like DAC could be customized with custom credentials.

```suggestion
While `DefaultAzureCredential` is generally the quickest way to authenticate apps for Azure, you can create a customized chain of credentials to be considered. `ChainedTokenCredential` enables users to combine multiple credential instances to define a customized chain of credentials. For more information, see [ChainedTokenCredential overview][ctc_overview].
```